### ISSUE TITLE
design audit: BAGPIPES↔tengri convention mismatch is a triangle (links #499 SED-side + #443 photometry-side)

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#515]
**Tracker for the convention-mismatch theme** uncovered while bundling the BAGPIPES-reproduction fixes in PR #501. Three issues all root in the same observation: tengri/DSPS and BAGPIPES make different convention choices at different layers of the SED→observable pipeline.

| Issue | Layer | Theme |
|---|---|---|
| **#499** | raw SED (`compute_csp_sed`) | 1.010 ± 0.001 flat ratio in optical (3000–10000 Å) |
| **#443** | band-integrated photometry (`predict_photometry`) | photon-Tokunaga vs photon-Bessell vs energy convention — 5–40 mmag band-dependent |
| #436 (closed via #446) | predict_magnitudes ↔ predict_photometry agreement | routed both through one convention (~0.07 mmag residual) |

The remaining open siblings (#499, #443) describe the same physics from two different layers — neither can be cleanly resolved alone, because picking a canonical convention at one layer leaks into the other.

## L_sun chain (relevant to #499)

| Path | L_sun [erg/s] |
|---|---|
| BC03 grid generator (Allen 1973) | 3.826e33 |
| BAGPIPES driver \`units.py:21\` (PR #493) | 3.826e33 |
| tengri \`compute_csp_sed\` via \`physics_constants.L_SUN\` | 3.828e33 (IAU 2015) |
| Cue training (kept distinct in \`L_SUN_CUE\`) | 3.839e33 |

Compounded across the read/integrate/multiply chain: max 0.5% — **cannot alone explain the flat 1.0% offset in #499**. There is at least one additional multiplicative factor somewhere in the SFH × SSP integration.

## The quadrature renormalisation footgun

In \`src/tengri/components/stellar/sps/dsps_wrapper.py:737\`:

\`\`\`python
total_mass = jnp.trapezoid(sfr_asc, t_cosmic_asc * 1e9)
age_weights_norm = joint.sum(axis=0)               # sum=1 per construction
age_weights_msun = age_weights_norm * total_mass
\`\`\`

This **exactly preserves** the SFH mass integral (explains why the BAGPIPES↔tengri \`massformed\` integrals agree to 0.03%), but lets per-bin quadrature error redistribute as a **flat multiplier on the SED**:

- 64 log-spaced bins (tengri default) integrating a delayed-tau SFH gives ~3.5% error vs the analytic truth.
- 218 linear bins (BAGPIPES) gives ~0.005% error.

A direct \`weights[i] = SFR[i] * dt[i]\` (no renormalisation) would expose the quadrature error in \`total_mass\` but make per-bin SED contributions faithful — the opposite tradeoff.

## Diagnostic plan for #499

1. **Isolate quadrature from SSP-scaling.** Run §3 with a delta-function age (single 5-Gyr SSP, no SFH integral). If the 1% persists → SSP / L_sun / port. If it vanishes → quadrature.
2. **Audit the L_sun chain end-to-end.** Verify there is no second Lsun multiplication anywhere in \`forward/pipeline.py\` or \`forward/sed_model.py\` that I missed.
3. **Compare BAGPIPES \`age_widths\` vs tengri DSPS-style joint weights at matched SFH.** A per-age weight comparison would localise the discrepancy.
4. **Spot-check both codes on a delta-SFR test.** If both produce identical SED for a single-burst SFH at age=5 Gyr, the SSP grids and L_sun chain match and the 1% is purely quadrature.

## Diagnostic plan for #443

Already documented in #443 itself (\`FilterConvention\` enum + SVO \`<DataType>\` detection).

## What we should NOT do

- **Don't \"fix\" #499 by ad-hoc tuning a multiplicative constant.** Pick the convention first.
- **Don't change \`L_SUN\` to match BAGPIPES.** IAU 2015 is the standard; BAGPIPES is the outlier.
- **Don't merge #499 + #443 into one PR without first writing an ADR** that picks the canonical convention.

## Suggested next action

Write an ADR (\`docs/adr/00NN-bagpipes-convention-audit.md\`) that:
1. Documents tengri's canonical choice (IAU 2015 L_sun, photon-Tokunaga / Bessell / energy — picked deliberately, not by inheritance).
2. Routes every layer through one convention.
3. Provides a single \`tengri.diagnostics.bagpipes_offset_audit()\` helper that runs the diagnostic plans above on any new SSP port.

Then file the implementation as a separate PR after the ADR lands.
